content/plugins/v1.0/_index.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Plugins Overview
+description: "plugins/README.md"
+date: 2020-11-02
+toc: true
+draft: false
+weight: 200
+cascade:
+  version: current
```

Comment:
Need to change the version here or the menu system won't work correctly.
```suggestion
  version: v1.0
```

Reply:
oh, good catch1